servers/display_server.h

```diff
@@ -399,7 +399,7 @@ class DisplayServer : public Object {
 	virtual void delete_sub_window(WindowID p_id);
 
 	virtual WindowID window_get_active_popup() const { return INVALID_WINDOW_ID; };
-	virtual void window_set_popup_safe_rect(WindowID p_window, const Rect2i &p_rect){};
```

Comment:
You could remove trailing `;` from other methods too.

Reply:
Done.;